_config.yml

```diff
@@ -23,5 +23,7 @@ navigation:
   url: documentation.html
 - text: Community
   url: contribute.html
+- text: Blog
+  url: blog.html
```

Comment:
Shouldn't this just link straight to the blog URL instead of having the redirect?  (The redirect may be useful for the sake of bookmarks/search engines, but not sure it's needed otherwise.)